batch_convert.qml

```diff
@@ -545,6 +555,7 @@ MuseScore {
     if (!inFormats.extensions.length)
       console.log("No input format selected")
 
+    if (outBB.checked)  outFormats.extensions.push("musicxml")
```

Comment:
So how is it defferent from the existing musicxml export?

Reply:
There is not difference, the Blackbinder format needs a musicxml as base, so the app can read it and make changes on it. The goal of adding the output is making it easier to the users, because they might not know what an xml is, and they can see directly the format they need for the app

Reply:
So there's no reason to add this option to this Plugin, as musicxml is supported already. Just educate your users that musixcxml is the format to use